src/Qir/.clang-tidy

```diff
@@ -1,5 +1,5 @@
 Checks: 
-  '-*,bugprone-*,-readability-*,readability-identifier-*,readability-braces-around-statements'
+  'bugprone-*,readability-identifier-*,readability-braces-around-statements,cert*'
```

Comment:
FYI: These checks in `.clang-tidy` file are added to the default ones: `clang-diagnostic-*,clang-analyzer-*`.
The `clang-diagnostic-*` enables the warnings of the Clang compiler.
The `clang-analyzer-*` enables the static analyzer. I.e. the Clang Static Analyzer is enabled by default, but our `.clang-tidy` file was disabling all the checks with the fragment `-*`. I removed `-*`, which enabled the Clang Static Analyzer.